apps/etterlatte-pdltjenester/src/main/kotlin/utils/MaskerFnr.kt

```diff
@@ -0,0 +1,3 @@
+package no.nav.etterlatte.utils
+
+fun String.maskerFnr(): String = if (this.length >= 6) this.substring(0..5) + "*****" else "*****"
```

Comment:
Kotlin har en fiffig måte å forenkle dette på. Brukes bl.a. i hjelpeklassen `Foedselsnummer` for å maskere de siste 5 sifferene i et personnr.

```
override fun toString(): String = this.value.replaceRange(6 until 11, "*****")
```